README.md

```diff
@@ -293,10 +287,25 @@ installed.
 1. [Build](#build) the changes and push them to some tag
 2. Use the new tag in your workflows:
 
-```diff
--uses: paritytech/pr-custom-review@1
-+uses: paritytech/pr-custom-review@2
-```
+    ```diff
+    -uses: paritytech/pr-custom-review@1
+    +uses: paritytech/pr-custom-review@2
+    ```
+
+### Deployment
+
+1. Create a token with `workflow`, `read:org`, `user` scopes.
+
+    ![Token scopes](./img/token-scopes.png)
```

Comment:
Why is "user" needed? Even `user:follow`?

Reply:
All those permissions should be justified. As in:

- `read:org` is needed in order to request reviews from teams
- `workflow` is neded because ...
- `user` is needed because ...

Reply:
You're right.
Orga scope is obvious, we should read teams (not sure about secret teams tho, but they can't be @mentioned so it's not really our use case)
Workflow is needed for the action to track the job's status.
And let me experiment with removing those user scopes altogether, I think it must work out.

Reply:
Just removed the `user` scope from the token. Restarted the job, [The output](https://github.com/paritytech/substrate/runs/5408901387?check_suite_focus=true) seems OK. Now need to fix a screenshot.